### PR TITLE
Add verifying Via, and trace version to system tests

### PR DESCRIPTION
**Pull request by pawelchcki**

### CODE REVIEW

**Comment by cbeauchesne** [comment]
https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md will be a good help :)

File: tests/test_telemetry.py [hunk 1]
```diff
@@ -21,13 +21,21 @@ def validator(data):
...
-    @missing_feature(library="python")
     def test_telemetry_proxy_enrichment(self):
         """Test telemetry proxy adds necessary information"""
         interfaces.agent.assert_headers_presence(
-            path_filter="/api/v2/apmtelemetry",
-            request_headers=["dd-agent-hostname", "dd-agent-env", "datadog-container-id"],
+            path_filter="/api/v2/apmtelemetry", request_headers=["dd-agent-hostname", "dd-agent-env"],
+        )
+        interfaces.agent.assert_headers_match(
+            path_filter="/api/v2/apmtelemetry", request_headers={"via": r"trace-agent 7\..+"},
```
Comment (pawelchcki) [inline tests/test_telemetry.py:30]: @cbeauchesne this test will fail on the current version of agent 7.36.x. 
IIRC there was an annotation, to mark failing releases. I'll look it up.

It is working correctly in version 7.35.x and below and it should be fixed in the version 7.37

Comment (cbeauchesne) [reply tests/test_telemetry.py:34]: the good annotation is something like @bug(context.agent >= "7.36.0" and  context.agent < "7.37.0")

**Review by cbeauchesne — APPROVED** [review]

**Comment by pawelchcki** [comment]
Thanks @cbeauchesne !

**Pull request merged**